Kitodo/src/main/java/org/kitodo/production/services/data/ImportService.java

```diff
@@ -1022,4 +1029,33 @@ public static void processTempProcess(TempProcess tempProcess, Template template
         addProperties(tempProcess.getProcess(), template, processDetails, docType, tempProcess.getProcess().getTitle());
         updateTasks(process);
     }
+
+    /**
+     * Imports a process and saves it to database.
+     * @param ppn the ppn to import
+     * @param projectId the projectId
+     * @param templateId the templateId
+     * @param selectedCatalog the selected catalog to import from
+     * @return the importedProcess
+     */
+    public Process importProcess(String ppn, int projectId, int templateId, String selectedCatalog) throws ImportException {
+        LinkedList<TempProcess> processList = new LinkedList<>();
+        Template template = null;
+        try {
+            template = ServiceManager.getTemplateService().getById(templateId);
+            String metadataLanguage = ServiceManager.getUserService().getCurrentUser().getMetadataLanguage();
+            List<Locale.LanguageRange> priorityList = Locale.LanguageRange.parse(metadataLanguage.isEmpty() ? "en" : metadataLanguage);
+            importProcessAndReturnParentID(ppn, processList, selectedCatalog, projectId, templateId);
+            processTempProcess(processList.get(0), template,
```

Comment:
This could probably lead to a null pointer exception if `processList.get(0)` is `null`.